docs/standard/library-guidance/nuget.md

```diff
@@ -46,7 +46,7 @@ A NuGet package supports many [metadata properties](/nuget/reference/nuspec). Th
 | `Description`                      | `description`              | A long description of the package displayed in UI.             |
 | `Authors`                          | `authors`                  | A comma-separated list of package authors, matching the profile names on nuget.org.             |
 | `PackageTags`                      | `tags`                     | A space-delimited list of tags and keywords that describe the package. Tags are used when searching for packages.             |
-| `PackageIconUrl`                   | `iconUrl`                  | A URL for an image to use as the icon for the package. URL should be HTTPS and the image should be 64x64 and have a transparent background.             |
+| `PackageIcon`                   | `icon`                  | A path to an image in the package to use as a package icon. Read more about [`icon` metadata](/nuget/reference/nuspec#icon). |
```

Comment:
@JamesNK, Does `PackageIconFullPath` need to be documented in this article as well? with an explanation whether it's needed together with `PackageIcon` or not?

Reply:
I opened an issue for that so that it doesn't get forgotten if needed to be documented.